templates/display_alert.html.twig

```diff
@@ -100,5 +99,14 @@
                     .replace(')', ', 0.6)')
             )
         }
+
+        // ITIL form has a specified layout, we must always display the full size
+        // alert.
+        // The "col-12" class can't be directly used for this purpose because
+        // a GLPI's core script will automatically replace it by "col-6" when
+        // expending the sidebar.
+        $('form[id="itil-form"]')
+            .find('.plugin_news_alert')
+            .addClass('w-100');
```

Comment:
The size defined in the alert parameters is no longer taken into account; it always takes up the full width.

A condition is probably missing

Reply:
This is done on purpose for ITIL items because we have not much available space in this case:

> ITIL form has a specified layout, we must always display the full size alert

Reply:
It should not happen anywhere else tho, where did you noticed it ?

Reply:
Adrien and I discussed this and wondered whether it wouldn't make more sense for the alert to take up the entire width of an ItilObject's information bar rather than leaving it small or medium-sized, as the text would become illegible.
(Of course, the size defined in the parameters does not change if we're not in an ItilObject form.)
**Before**
![image](https://github.com/pluginsGLPI/news/assets/107540223/49a0f099-16fd-43a1-801d-d2db6660f637)
**After**
![image](https://github.com/pluginsGLPI/news/assets/107540223/9cc81685-5326-4b4e-a2e8-1531f347f83b)

Reply:
I've seen it on tickets.

Indeed, it's not relevant here that the maximum width is not always taken into account.

But if we were to change this behavior, I'm afraid it would be considered a bug. In this case, we should specify in the alert parameters around the "Size" field.

![image](https://github.com/pluginsGLPI/news/assets/8530352/eb9f848b-707f-411c-a335-2aba95879f44)

Reply:
I guess if you really want to inform the user then we can have a tooltip after "Size" with the relevant info.